src/Compilers/Core/Portable/Hashing/XxHashShared.cs

```diff
@@ -342,7 +342,7 @@ public static void CopyAccumulators(ref State state, ulong* accumulators)
         {
             fixed (ulong* stateAccumulators = state.Accumulators)
             {
-#if NET7_0_OR_GREATER
+#if NET8_0_OR_GREATER
```

Comment:
same with these checks.  the apis we're gating around are on .net7.  this would mean we would fall back to the lower performance code there.

Reply:
> the apis we're gating around are on .net7. this would mean we would fall back to the lower performance code there.

We no longer target net7.